go/host/p2p/p2p.go

```diff
@@ -260,31 +262,45 @@ func (p *p2pImpl) send(msg message, to string) error {
 	if err != nil {
 		return fmt.Errorf("could not encode message to send to sequencer. Cause: %w", err)
 	}
-	p.sendBytes(nil, to, msgEncoded)
+	err = p.sendBytesWithRetry(nil, to, msgEncoded)
+	if err != nil {
+		return err
+	}
 	return nil
 }
 
 // Sends the bytes to the provided address.
-func (p *p2pImpl) sendBytes(wg *sync.WaitGroup, address string, tx []byte) {
+// Until introducing libp2p (or equivalent), we have a simple retry
+func (p *p2pImpl) sendBytesWithRetry(wg *sync.WaitGroup, address string, msgEncoded []byte) error {
 	if wg != nil {
 		defer wg.Done()
 	}
+	// retry for about 2 seconds
+	err := retry.Do(func() error {
+		return p.sendBytes(address, msgEncoded)
+	}, retry.NewDoublingBackoffStrategy(100*time.Millisecond, 5))
+	return err
```

Comment:
If we swallow the error when returned, lets log it here if possible with a warning

Reply:
it is logged as warning already in "sendBytes"